package.json

```diff
@@ -357,6 +357,13 @@
               "Show the Swift build status as a progress notification."
             ],
             "order": 17
+          },
+          "swift.warnAboutSymlinkCreation": {
+            "type": "boolean",
+            "default": true,
+            "description": "Controls whether or not the extension will warn about being unable to create symlinks. (Windows only)",
+            "scope": "application",
+            "order": 18
```

Comment:
Why is this an option? The extension will not work correctly if symlink creation is disabled? So we should warn about it regardless

Reply:
In my testing the extension mostly works even with symbolic links disabled. I'm able to build a simple application and use SourceKit-LSP for language features. The only thing I noticed is that the following warning is displayed by swift while building:
```
warning: unable to create symbolic link at C:\Users\matthew_bastien\Documents\swift-executable\.build\debug: Error Domain=NSCocoaErrorDomain Code=256 "(null)"
```

Of course, this means that debugging won't work, but you still get other features and can `swift run` from the command line. I can see a circumstance in which a user might not care (or be unable to modify the policy for whatever reason) and just want the warning to not show up every time they open a swift project.